scripts/babel-relay-plugin/src/RelayQLAST.js

```diff
@@ -709,6 +709,37 @@ class RelayQLType {
     };
     return new RelayQLFragment(this.context, (generatedFragmentAST: any), this);
   }
+
+  getNodeIdField(): ?string {
+    const node = this.context.schema._typeMap['Node'];
+    if (node && node instanceof types.GraphQLInterfaceType) {
+      const fields = node._fields;
```

Comment:
Never mind. Found it and obvious in hindsight (I’m not up-to-date yet with all JS conventions).